### DISCUSSION TITLE
Accidentally deleted user 'consumer' - Can't add new documents

### DISCUSSION THREAD

**Discussion by feerlessleadr** [jonaswinkler/paperless-ng#1164] [Support] [answered]
So in a moment of complete idiocy, I accidentally deleted the 'consumer' user from the admin panel without realizing, and now I'm unable to add any new documents to my paperless-ng instance. 

Is there any way that I can re-add this user? I searched the github and documentation, but don't see any steps on the best way to recover from this brain-dead move. 

I do have nightly backups of the pgdata and data docker volumes, so worst case I think I can just roll back to my last backup that has the user still enabled, but I was hoping not to do that, since I've added a bunch of documents since thing, and didn't want to have to re-add if I can avoid it.

Thanks

**Comment by jonaswinkler** [comment, marked as answer]
You can also use the admin panels to add users to the system again. Just add a "consumer" user, it should fix the situation.

**Reply by feerlessleadr** [reply]
Thanks, I thought that might be it, although it required me to add a password to the account. I thought that the original account didn't have a password, so (perhaps incorrectly) assumed that there was a different way to do it. 

In that case, should I make the password something random (I can use my password manager for that if needed)?

**Reply by feerlessleadr** [reply]
disregard that - I just re-created the consumer account with a random password and everything is working again. Thanks for the solve!
